engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EnsoProjectNode.java

```diff
@@ -126,7 +126,7 @@ public Object getOtherProjectDescr(
 
   private static Atom createProjectDescriptionAtom(EnsoContext ctx, Package<TruffleFile> pkg) {
     EnsoFile rootPath = new EnsoFile(pkg.root().normalize());
-    Object cfg = ctx.getEnvironment().asGuestValue(pkg.getConfig());
+    Object cfg = ctx.asGuestValue(pkg.getConfig());
```

Comment:
Future work: Exposing `pkg.getConfig()` is wild idea - we should create an `EnsoObject` subclass and expose the needed members via `@ExportMessage`.